### PR TITLE
Introduce standalone TBB

### PR DESCRIPTION
**Pull request by sawenzel**
Build ROOT against standalone TBB (Intel Thread Building Blocks)
and make TBB available for other packages.

### CODE REVIEW

**Comment by ktf** [comment]
@TimoWilken could you warm up the cache with this?

File: tbb.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+cmake --build . -- ${JOBS:+-j$JOBS} install
+
+# make soft link of lib64 dir to lib
+pushd "$INSTALLROOT"
+  # Fix ambiguity between lib/lib64
+  if [[ ! -d lib && -d lib64 ]]; then
+    ln -nfs lib64 lib
+  elif [[ -d lib && ! -d lib64 ]]; then
+    ln -nfs lib lib64
+  fi
```
Comment (ktf) [inline tbb.sh:27]: Why not simply using `-DCMAKE_INSTALL_LIBDIR=lib`?

Comment (sawenzel) [reply tbb.sh:27]: no idea. I took the inspiration from different recipes in alidist

Comment (ktf) [reply tbb.sh]: Thank you for your update. Indeed we should probably fix also the other recipes.

**Review by ktf — APPROVED** [review]

**Comment by TimoWilken** [comment]
Caching is done.

**Pull request merged**